lib/ansible/modules/package.py

```diff
@@ -44,6 +44,25 @@
     default: auto
 requirements:
     - Whatever is required for the package plugins specific for each system.
+extends_documentation_fragment:
+  -  action_common_attributes
+  -  action_common_attributes.flow
+attributes:
+    action:
+        support: full
+    async:
+        support: full
+    bypass_host_loop:
+        support: none
+    check_mode:
+        details: support depends on the underlying plugin invoked
+        support: N/A
```

Comment:
And here we have a place where IMO `unknown` would be much better than `N/A`. Not applicable means that check mode makes no sense for the package action, which isn't true - the correct answer is "we do not know because there are multiple backends", which isn't exactly "unknown", but "unknown" comes much closer to the correct answer IMO. :)